tests/models/grounding_dino/test_modeling_grounding_dino.py

```diff
@@ -741,3 +780,53 @@ def test_cross_attention_mask(self):
         self.assertTrue(torch.allclose(outputs1.logits, outputs_batched.logits[:1], atol=1e-3))
         # For some reason 12 elements are > 1e-3, but the rest are fine
         self.assertTrue(torch.allclose(outputs2.logits, outputs_batched.logits[1:], atol=1.8e-3))
+
+    def test_grounding_dino_loss(self):
+        ds = load_dataset("EduardoPacheco/aquarium-sample", split="train")
```

Comment:
Maybe we should move this to huggingface?

Reply:
Maybe, wdyt @amyeroberts

Reply:
Thanks for working on this, is great to have it tested!
Do we actually need to load the whole dataset here? Can't we copy annotations and upload one image somewhere?

Reply:
yes, no, maybe?

Reply:
@ydshieh can you please help, do we have any place for test assets?

Reply:
@EduardoPach @qubvel I think we can do like this
https://github.com/huggingface/transformers/blob/be9cf070ee2cb6a9f0d162e5be32d9d68b9df3af/tests/models/detr/test_modeling_detr.py#L562

Reply:
Hi. If IIRC, that dataset contains only 2 examples, right?

The easiest way to have it as 

> hf-internal-testing/aquarium-sample

Reply:
@qubvel  I added you to `hf-internal-testing` (you have to accept the invitation) then you can create a copy of the above dataset.